migrations/versions/47a38ad4ce75_.py

```diff
@@ -0,0 +1,33 @@
+"""empty message
```

Comment:
try to remember to add a message when you create a migration. It can make it easier to know which migration files to look in if there's a problem going on during setup.